README.md

```diff
@@ -1,25 +1,33 @@
-Picnic Technologies Open Source Website
-=========================
+# Picnic Technologies Open Source Website
 
 Showcasing our latest open-source projects.
 
 Live at: [http://picnic.tech](http://picnic.tech)
 
 ## How to build locally
-1. Make sure you have bundler installed, if not run `gem install bundler`
-2. `> git clone git@github.com:PicnicSupermarket/picnicsupermarket.github.io.git`
-3. `> cd picnicsupermarket.github.io`
-4. Run `bundle install` to install all dependencies
-5. `> jekyll serve`
-6. View at `http://127.0.0.1:4000`
+
+```sh
+gem install bundler # If bundler is not installed
+git clone git@github.com:PicnicSupermarket/picnicsupermarket.github.io.git
+cd picnicsupermarket.github.io
+bundle install
+bundle exec jekyll serve --livereload
+```
+
+You can view the preview at `http://127.0.0.1:4000`
 
 ## How to add a new project
-1. Create a new file in the `_posts` folder which follows the naming conventions.
-2. The mandatory fields are: `layout`, `title`, `technology`, `image`, `desc`, `proj-url`, `proj-num`.
-3. If this is a "Coming Soon" project, use the `coming-soon.png` image. Otherwise, drop a new file in the `images` folder and simply reference it by its name.
-4. Regenerate the site using `jekyll build --future`.
+
+1. Create a new file in the `_posts` folder which follows the naming
+   conventions: `year-month-date-project-name`
+2. The mandatory fields are: `layout`, `title`, `technology`, `image`, `desc`,
+   `proj-url`, `proj-num`.
+3. If this is a "Coming Soon" project, use the `coming-soon.png` image.
+   Otherwise, drop a new file in the `images` folder and simply reference it by
+   its name.
 
 ## Built using
-+ [GitHub Pages](https://pages.github.com/)
-+ [Jekyll](https://jekyllrb.com/)
-+ [Jekyll-grid theme](http://femmebot.github.io/jekyll-grid)
+
+- [GitHub Pages](https://pages.github.com/)
+- [Jekyll](https://jekyllrb.com/)
+- [Jekyll-grid theme](http://femmebot.github.io/jekyll-grid)
```

Comment:
Same thing. :+1: 
```suggestion
- [Jekyll-grid theme](https://femmebot.github.io/jekyll-grid)
```

Reply:
We can, although that site doesn't have a certificate and Chrome tell me it's not secure.

Reply:
Works well on my Chrome
![image](https://user-images.githubusercontent.com/12081293/193248436-ccfb74a3-1e0f-4f22-ac18-e9f544618e22.png)

But I saw that you had already updated it. :+1: